vault/data_source_auth_backends.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+)
+
+func authBackendsDataSource() *schema.Resource {
+	return &schema.Resource{
+		Read: ReadWrapper(authBackendsDataSourceRead),
+		Schema: map[string]*schema.Schema{
+			consts.FieldPaths: {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Description: "The auth backend mount points.",
+			},
+			consts.FieldType: {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The type of the auth backend.",
+			},
+			consts.FieldAuthMethodAccessors: {
```

Comment:
I prefer the name `accessors` since this name stutters:

`data.vault_auth_backends.test.auth_method_accessors`

vs

`data.vault_auth_backends.test.accessors`

Also the existing `vault_auth_backend` resource uses `accessor` so I think it makes sense to be similar to that.